pkg/util/pravegacluster.go

```diff
@@ -64,6 +64,10 @@ func HealthcheckCommand(port int32) []string {
 	return []string{"/bin/sh", "-c", fmt.Sprintf("netstat -ltn 2> /dev/null | grep %d || ss -ltn 2> /dev/null | grep %d", port, port)}
 }
 
+func ReadinessHealthcheckCommand(port int32) []string {
```

Comment:
Would be good to call this method `ControllerReadinessCheck()` to indicate its appropriate purpose....

Reply:
I have changed it.

Reply:
ok.